tools/testoperator/dispatch/tpch/sf1/accelerated/dynamodb-duckdb[file].yaml

```
@@ -4,17 +4,17 @@ tests:
     query_set: tpch
     runner_type: spiceai-dev-runners
     ready_wait: 600
-    query_overrides: dynamodb-accelerated
+    query_overrides: dynamodb
   throughput:
     spicepod_path: accelerated/dynamodb-duckdb[file].yaml
     query_set: tpch
     runner_type: spiceai-dev-runners
     ready_wait: 600
-    query_overrides: dynamodb-accelerated
+    query_overrides: dynamodb
```

use the accelerator specific query overrides, or add a comment why this is required.


```suggestion
    query_overrides: duckdb
```